Makefile.common

```diff
@@ -175,15 +175,38 @@ common-tarball: promu
 
 .PHONY: common-docker
 common-docker:
-	docker build -t "$(DOCKER_REPO)/$(DOCKER_IMAGE_NAME):$(DOCKER_IMAGE_TAG)" .
+	docker build -t "$(DOCKER_REPO)/$(DOCKER_IMAGE_NAME):$(DOCKER_IMAGE_TAG)" \
+		--build-arg BASE_IMAGE="quay.io/prometheus/busybox:latest" \
+		--build-arg PROMETHEUS_PATH=".build/linux-amd64/prometheus" \
```

Comment:
This won't work for other repos, Makefile.common is used elsewhere too.

Reply:
This would require changes to all other repos too, that's right. But I still feel it's reasonable to do that.

Reply:
That's the whole point of Makefile.common, we copy it directly to other repos.

Reply:
I'm happy to change it to whatever direction you prefer, but I'm currently not sure whom to follow.

The building itself could we just multiple calls to `make docker` with `BASE_IMAGE`, `BUILD_DIR`  and `DOCKER_IMAGE_NAME` as parameters. Building the manifest itself might be a bit more tricky as it requires knowledge of all included architectures, but nothing unsolvable.

Reply:
@brian-brazil I pushed https://github.com/prometheus/node_exporter/pull/1207 with much more make "magic" to dynamically generate targets based on a per projects `DOCKER_ARCHS` variable, let me know whether which approach you prefer.

Reply:
@simonpasquier and @SuperQ are the ones more looking after this area.

Reply:
@johanneswuerbach The goal here is to have the Makefile.common work without changes for all prometheus repositories, so we can't hardcode build args that are specific to a repo (like PROMETHEUS_PATH).

Not sure what the best way to solve this would be. Something like `BIN_PATH` in prometheus/node_exporter#1207 might work. Somebody would need to make sure this works for all other repos.